demo-apps/angular/angular-app/src/index.html

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>AngularApp</title>
+  <base href="/">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.min.css" crossorigin="anonymous">
```

Comment:
Add UF?

Reply:
I was just hoping there'd be a simple example, and chose to omit the UF for now. But I'll see if I have time to add it in today!

Reply:
I did find time today, yay! Added the UF.